PowerPointLabs/PowerPointLabs/TextCollection.cs

```diff
@@ -18,161 +18,197 @@ public class TextCollection
+        public const string EffectsLabMakeTransparentSupertip =
+            "Adjust the transparency of pictures or shapes." +
+            "To perform this action, select the shape(s) or picture(s), then click this button.";
+        public const string EffectsLabMagnifyGlassSupertip =
+            "Magnify a small area or detail on the slide.\n\n" +
+            "To perform this action, select the shape over the area to magnify, then click this button.";
+        public const string EffectsLabBlurSelectedSupertip =
+            "Blur the area covered by the selected shapes.\n\n" +
+            "To perform this action, select the shape(s) over the area to blur, then click this button.";
+        public const string EffectsLabBlurRemainderSupertip =
+            "Blur evrything in the slide except for the area covered by the selected shapes.\n\n" +
+            "To perform this action, select the shape(s) over the area to keep, then click this button.";
+        public const string EffectsLabBlurBackgroundSupertip =
+            "Blur everything in the slide except for the selected shapes." +
+            "To perform this action, select the shape(s) or picture(s) to keep, then click this button.";
+        public const string EffectsLabColorizeRemainderSupertip =
+            "Recolor an area of a slide to attract attention to it." +
+            "To perform this action, select the shape(s) over the area to keep, then click this button.";
+        public const string EffectsLabColorizeBackgroundSupertip =
+            "Recolor everything in the slide except for the selected shapes." +
+            "To perform this action, select the shape(s) or picture(s) to keep, then click this button.";
         # endregion
 
         # region Agenda Lab
-        public const string AgendaLabSupertip = "Generate professional-look agenda automatically.\n\n To use this feature, you need " +
-                                                "to group up your into appropriate sections. Each section will be used as one item in " +
-                                                "the agenda.";
-        public const string AgendaLabBulletPointSupertip = "Generate an agenda in bullet point style.";
-        public const string AgendaLabVisualAgendaSupertip = "Generate an agenda in visual style.";
-        public const string AgendaLabBeamAgendaSupertip = "Generate agenda side bar for selected slides.";
-        public const string AgendaLabUpdateAgendaSupertip = "Synchronize agenda's layout and format with the first slide.";
-        public const string AgendaLabRemoveAgendaSupertip = "Remove agenda generated by PowerPointLabs.";
-        public const string AgendaLabAgendaSettingsSupertip = "Configure agenda settings.";
-        public const string AgendaLabBulletAgendaSettingsSupertip = "Set color scheme for Bullet Agenda.";
+        public const string AgendaLabSupertip =
+            "Generate professional-look agenda automatically.\n\n" +

```

Comment:
To be consistent, maybe "Use Agenda Lab to automatically create professional-looking agenda slides.".